src/table/utils/__tests__/handle-accessiblity.spec.ts

```diff
@@ -157,31 +162,31 @@ describe('handle-accessibility', () => {
 
     it('should indirectly call updateFocus, setFocusedCellCoord and keyboard.focus', () => {
       handleAccessibility.handleClickToFocusHead(columnIndex, rootElement, setFocusedCellCoord, keyboard);
-      expect(cell.setAttribute).toHaveBeenCalledTimes(1);
-      expect(cell.setAttribute).toHaveBeenCalledWith('tabIndex', '-1');
+      expect(cell?.setAttribute).toHaveBeenCalledTimes(1);
+      expect(cell?.setAttribute).toHaveBeenCalledWith('tabIndex', '-1');
       expect(setFocusedCellCoord).toHaveBeenCalledTimes(1);
       expect(setFocusedCellCoord).toHaveBeenCalledWith([0, 2]);
       expect(keyboard.focus).toHaveBeenCalledTimes(1);
     });
   });
 
   describe('handleMouseDownLabelToFocusHeadCell', () => {
-    const evt = { preventDefault: jest.fn() };
+    evt = { preventDefault: jest.fn() } as unknown as BaseSyntheticEvent;
     const columnIndex = 0;
 
     it('should indirectly call updateFocus, setFocusedCellCoord and keyboard.focus', () => {
       handleAccessibility.handleMouseDownLabelToFocusHeadCell(evt, rootElement, columnIndex);
       expect(evt.preventDefault).toHaveBeenCalledTimes(1);
-      expect(cell.focus).toHaveBeenCalledTimes(1);
-      expect(cell.setAttribute).toHaveBeenCalledTimes(1);
-      expect(cell.setAttribute).toHaveBeenCalledWith('tabIndex', '0');
+      expect(cell?.focus).toHaveBeenCalledTimes(1);
+      expect(cell?.setAttribute).toHaveBeenCalledTimes(1);
+      expect(cell?.setAttribute).toHaveBeenCalledWith('tabIndex', '0');
     });
   });
 
   describe('handleResetFocus', () => {
-    let shouldRefocus;
-    let isSelectionMode;
-    let announce;
+    let shouldRefocus: React.MutableRefObject<boolean>;
+    let isSelectionMode: boolean;
+    let announce: AnnounceFn;
```

Comment:
AnnounceFn is Announce in the main branch. Please merge the main branch to your branch.

Reply:
Sorry, didn't get what you mean? It's already available in types in my branch ...

Reply:
I mean that your branch needs to be in sync with the main branch. In the main branch, there is no AnnounceFn anymore.

Reply:
Aha in main it is still Announce. I'll fix it.

Reply:
Thanks!